searchlib/pom.xml

```diff
@@ -51,10 +51,18 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
+      <exclusions>
```

Comment:
Note: this exclusion is unnecessary

Reply:
Agreed, but it is done most places this dependency is added. I suggest someone make a PR which removes it everywhere.